generated/python/grpc-google-service-logging-types-v1/setup.py

```diff
@@ -0,0 +1,43 @@
+"""A setup module for the GRPC Stackdriver Logging External Types service.
+
+See:
+https://packaging.python.org/en/latest/distributing.html
+https://github.com/pypa/sampleproject
+"""
+
+import setuptools
+
+from setuptools import setup, find_packages
+
+install_requires = [
+  'oauth2client>=2.0.0, <4.0dev',
```

Comment:
We should start setting the upper-bound at 5.0dev, but that can be done across-the-board in a separate PR.